apps/mobile/src/features/send/send-form/send-form.schema.ts

```diff
@@ -0,0 +1,19 @@
+import { z } from 'zod';
+
+export const sendFormSchema = z.object({
+  amount: z.string().refine(val => !isNaN(Number(val)) && Number(val) > 0, {
+    message: '',
+  }),
+  recipient: z.string(),
+  memo: z.string().optional(),
+  fee: z.string(),
+});
+
+export type SendFormSchema = z.infer<typeof sendFormSchema>;
+
+export const defaultSendFormValues: SendFormSchema = {
+  amount: '',
+  recipient: '',
+  memo: '',
+  fee: '',
+};
```

Comment:
How might we auto-set a recipient if there's ever a feature like "send again" somewhere in the UI?